src/NuGet.Commands/Strings.resx

```diff
@@ -151,7 +151,7 @@
     <value>Some packages are not compatible with {0}.</value>
   </data>
   <data name="Log_ProjectDoesNotSpecifyTargetFrameworks" xml:space="preserve">
-    <value>The project does not specify any target frameworks.</value>
+    <value>The project {0} does not specify any target frameworks in project.json.</value>
```

Comment:
Add single quotes around the project name like this '{0}'   It makes it easier to understand the message when the project has an odd name, or a name that blends into the sentence.